dev/devicelab/bin/tasks/build_aar_module_test.dart

```diff
@@ -149,9 +177,9 @@ Future<void> main() async {
         'flutter',
         'devicelab',
         'hello',
-        'flutter_release',
```

Comment:
how did this pass before? Should we clean somewhere in here to make sure it's hermetic-ish?

Reply:
The test built the release version before the debug version :)